### DISCUSSION TITLE
Feature request: Flatpak package.

### DISCUSSION THREAD

**Discussion by ForeverNooob** [jcorporation/myMPD#824] [Ideas]
Hi,

As of now there is no Flatpak app of myMPD available, which is a bit unfortunate for me since I'm on a pretty old Ubuntu 18.04 install right now and as such can't easily install myMPD.

Open Build Service appears to supports Flatpak, so the support could probably be added trivially: https://openbuildservice.org/2021/02/18/introducing-flatpak-builds/

Thanks in advance for considering it.

**Comment by jcorporation** [comment]
From https://docs.flatpak.org/en/latest/introduction.html:

> Flatpak is a framework for distributing desktop applications

myMPD is a server application. You could use docker for installation: https://jcorporation.github.io/myMPD/installation/docker

**Comment by ForeverNooob** [comment]
But it's also a webapp, right? So a web browser is required. You could include a browser in the Flatpak build (like Electron) and that could work I suppose.

I personally don't like the way Docker messes with my firewall settings, so I generally avoid that on my desktop.

**Reply by jcorporation** [reply]
Why do you not upgrade Ubuntu to a recent version and use the Debian package?

**Reply by ForeverNooob** [reply]
> Why do you not upgrade Ubuntu to a recent version and use the Debian package?

Yeah that would be ideal, but I have too much things I need to fix first (unrelated to mpd and such) before a headache-free upgrade.
